tests/unit_tests/variant.cpp

```diff
@@ -26,41 +26,45 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#include <sstream>
-#include <type_traits>
-#include <vector>
+#include "common/variant.h"
 
 #include <boost/mpl/deref.hpp>
 #include <boost/variant/recursive_wrapper.hpp>
 #include <boost/variant/recursive_variant.hpp>
 
 #include "gtest/gtest.h"
 
-#include "common/variant.h"
+#include <sstream>
+#include <type_traits>
+#include <vector>
 
 using tools::variant;
 using tools::variant_static_visitor;
 
 namespace
 {
```

Comment:
I still think the anonymous namespace is redundant, but leaving it in here since it's your file.

Reply:
I particularly like using anonymous namespaces because `static` doesn't prevent ODR violations. I don't know, I don't care too much either way.